defaults/main.yml

```diff
@@ -48,7 +48,8 @@ tiaas_other_config: ""
 tiaas_info:
   owner: default
   owner_email: admin@localhost
-  owner_site: "{{ inventory_hostname }}"
+  owner_site: "http://{{ inventory_hostname }}"
+  url: "http://{{ inventory_hostname }}"
```

Comment:
They will, but they should redirect to HTTPS in that case? AU does at least.
This will likely change anyway, I got in a knot over these variables cause AU currently has `domain: usegalaxy.org.au` when the tiaas application actually expects a URL:

```py
# local_settings.py.template
TIAAS_DOMAIN = "https://galaxy.example.org" # Your Galaxy instance
```

I think it makes more sense to just expect a domain name and built the URL as a separate variable in `settings.py`. What do you think? Again this will be mildly annoying for admins that have set `domain` to a URL but I think only EU has done that. It's not a variable that's used much in the code so the impact is minor anyway.

[Galaxy main conf](https://github.com/galaxyproject/usegalaxy-playbook/blob/main/env/common/group_vars/tiaasservers/vars.yml):
```yml
tiaas_info:
  owner: "{{ galaxy_instance_hostname }}"
  owner_email: galaxy-bugs@galaxyproject.org
  owner_site: "https://{{ galaxy_instance_hostname }}/"
  domain: "{{ galaxy_instance_hostname }}"
```

[Galaxy AU conf](https://github.com/usegalaxy-au/infrastructure/blob/master/host_vars/aarnet.usegalaxy.org.au.yml#L230):
```yml
tiaas_info:
  owner: "Galaxy Australia"
  owner_email: help@genome.edu.au
  owner_site: "https://usegalaxy.org.au"
  domain: "usegalaxy.org.au"
```

Reply:
no if it says "domain", it should be "domain", I agree with that. Ok, sounds good. Let's do it this way.

Reply:
Have updated tiaas with this:

```py
GALAXY_DOMAIN = "galaxy.example.org"  # Your Galaxy hostname

# After local_settings import
GALAXY_URL = f"http://{GALAXY_DOMAIN}"  # This is used in several templates
```

Still unsure about the `http` bit - if anyone has deployed with certbot it should have added the http redirect so I think this should be fully compatible. Then again, I don't know why anyone would deploy without certbot!